src/components/form-file/form-file.js

```diff
@@ -152,6 +156,11 @@ export default {
       } else {
         this.$emit('input', newVal)
       }
+    },
+    value (newVal) {
+      if (newVal === null || newVal === '') {
```

Comment:
What is about empty arrays when `multiple` is enabled?
Is there any particular reason this isn't taken into account here?

If not, I would suggest to change the check to:
```js
if (
  (this.multiple && Array.isArray(newVal) && newVal.length === 0) ||
  (!this.multiple && (newVal === null || newVal === ''))
) {
  ...
}
```

Reply:
i thought about that (and originall had it typed in similar), but figured `null` is ideally the best (and it's the default intenally). although it might be a good addition, just in case. feel free to push the change to this pr :)

Reply:
actually, I think you could skip the check for multiple and just check of it is an array that is zero length, or null or empty string. no need to make the tests too complex.

Reply:
also, when the input is reset, we send out an update to the v-model to update to either an empty array (in multiple mode) or null if in single mode

Reply:
for the isArray test, we could pull in isArray from utils/array.js, as it helps with minification.

Reply:
@tmorehouse Wouldn't it be easier if you apply the changes? :)

Reply:
lol, I was out getting groceries..